src/utils/validation/registration/contributorValidation.ts

```diff
@@ -5,37 +5,41 @@ import { ErrorMessage } from '../errorMessage';
 
 const contributorValidationSchema = Yup.object().shape({
   correspondingAuthor: Yup.boolean(),
+  sequence: Yup.number(),
+  role: Yup.string(),
   email: Yup.string().when('correspondingAuthor', {
     is: true,
     then: Yup.string().email(ErrorMessage.INVALID_FORMAT).required(ErrorMessage.REQUIRED),
   }),
 });
 
-export const contributorsValidationSchema = Yup.array()
-  .when('$publicationInstanceType', {
-    is: BookType.ANTHOLOGY,
-    then: Yup.array()
-      .of(contributorValidationSchema)
-      .test('editor-test', ErrorMessage.MISSING_EDITOR, (contributors) =>
-        hasRole(contributors, ContributorRole.EDITOR)
-      ),
-    otherwise: Yup.array()
```

Comment:
syns denne ble litt ekkel, for hvis det er en Degree vil `is: BookType.Anthology` bli `false`, så man ender i otherwise her. Men man vil jo også ende opp i `is: PublicationType.Degree` i when som ligger under. Skjønner liksom ikke helt om det da er begge som gjelder, eller bare siste eller hva det er. Så selv om ny versjon har litt mer knotete syntaks syns jeg det gir mer mening når man leser det

Reply:
ja, det er siste som gjelder. syntes selv det var litt rart, så skjønner at du ble forvirret ja